packages/wallet/ui/src/components/ProvisionDialog.jsx

```diff
@@ -96,7 +141,18 @@ const ProvisionDialog = ({ onClose, open, address, href, keplrConnection }) => {
       default:
         return <></>;
     }
-  })();
+  }, [currentStep, href, address]);
+
+  console.log('provisionPoolData', provisionPoolData);
+  const provisionPoolLow = useMemo(
+    () =>
+      provisionPoolData &&
+      AmountMath.subtract(
+        provisionPoolData.totalMintedConverted,
+        provisionPoolData.totalMintedProvided,
+      ).value < MINIMUM_PROVISION_POOL_BALANCE,
+    [provisionPoolData],
+  );
```

Comment:
this is a very cheap calculation that's probably not worth memoizing.

also it's outside the rendering domain so consider pulling out into a non-React function. maybe even a different module that has the constant too. that could be unit tested.

Reply:
Pulled into a different function but not a module for now. The wallet UI still uses jest so we can't use ses and hence can't test any amountmath stuff without mocking a lot.